include/surface_perception/segmentation.h

```diff
@@ -173,15 +180,17 @@ bool GetSceneAboveSurface(pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud,
 ///   cluster for that cluster to be considered an object.
 /// \param[in] max_cluster_size The maximum number of points that can be in a
 ///   cluster for that cluster to be considered an object.
-/// \param[out] surface_objects The surface and the objects above it that were
+/// \param[out] surfaces_objects_vec The vector of surfaces and the objects
+/// above each surface that were
 ///   found.
 ///
 /// \returns true if the segmentation was successful, false otherwise.
 bool FindObjectsOnSurface(pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud,
```

Comment:
This should be renamed FindObjectsOnSurfaces, since it operates on more than one surface.

Reply:
Completed